theano/compile/profilemode.py

```diff
@@ -695,6 +695,29 @@ def exp_float32_op(op):
         if not printed_tip:
             print("  Sorry, no tip for today.")
 
+    def clone(self, link_kwargs=None, message=None):
```

Comment:
instead of maintaining ProfileMode, why not delete it now? This would make testing this PR easier.